js/misc/paygManager.js

```diff
@@ -309,15 +325,30 @@ var PaygManager = GObject.registerClass({
             log("Unable to validate PAYG code: no regex")
             return false;
         }
+        
+        let is_match, match_info;
+
+        if (partial)
+            [is_match, match_info] = this._codeFormatRegex.match(Main.paygManager.codeFormatPrefix + code, 0);
+        else
+            [is_match, match_info] = this._codeFormatRegex.match(Main.paygManager.codeFormatPrefix + code +
+                                                                     Main.paygManager.codeFormatSuffix, 0);
 
-        let [is_match, match_info] = this._codeFormatRegex.match(code, 0);
         return is_match || (partial && match_info.is_partial_match());
     }
 
     get initialized() {
         return this._initialized;
     }
 
+    get codeFormatPrefix() {
+        return this._codeFormatPrefix;
```

Comment:
The idea was just to follow convention, because the paygmanager never returns info directly from the proxy, it always save on a local variable (e.g. ratelimitendtime and enabled). Of course, those two emits signals anytime they are changed.

If that's okay, I can just return them and get rid of the the setters too.

Reply:
Fine by me either way. (I didn't write the original code.)